src/databricks/labs/ucx/cli.py

```diff
@@ -11,9 +11,9 @@
 from databricks.labs.ucx.framework.tui import Prompts
 from databricks.labs.ucx.hive_metastore import ExternalLocations, TablesCrawler
 from databricks.labs.ucx.hive_metastore.mapping import TableMapping
-from databricks.labs.ucx.hive_metastore.tables import TablesMigrate
 from databricks.labs.ucx.install import WorkspaceInstaller
 from databricks.labs.ucx.installer import InstallationManager
+from databricks.labs.ucx.migration.table_migrate import TablesMigrate
```

Comment:
```suggestion
from databricks.labs.ucx.hive_metastore.migration import TablesMigrate
```

keep the `hive_metastore` package, don't introduce the `migration` package.

Reply:
OK, scrapping this idea.
It thought to keep it on par with "Assessment"